proposals/rkt.adoc

```diff
@@ -0,0 +1,81 @@
+## rkt
+
+*Name of project*: rkt
+
+*Description*:
+
+rkt is an application container engine developed for modern production cloud-native environments. It features a pod-native approach, a pluggable execution architecture, and a well-defined surface area that makes it ideal for integration with other systems.
+
+The core execution unit of rkt is the _pod_, a collection of one or more applications executing in a shared context. (rkt's pods are synoymous with the concept in the Kubernetes orchestration system). rkt allows users to apply different configurations (like isolation parameters) at both pod-level and at the more granular per-application level. rkt's architecture means that each pod executes directly in the classic Unix process model (i.e. there is no central daemon), in a self-contained, isolated environment.
```

Comment:
This probably doesn't matter for this proposal but for the sake of clarification...
I'm not a fan of "pod-native" because it's meaningless without a background and understanding of what a pod means in Kubernetes. Even then, many people who run Kubernetes I don't use multi-container deployments and just think of a pod as a container.

I would probably reword it to say something like multi-container application deployments or multi-container bundles. Something that's more descriptive and not as abstract.

I'm sure everyone reading this proposal knows exactly what is meant by a "pod" but that term caused confusion for the other descriptions of rkt.